lquota

```diff
@@ -116,9 +116,15 @@ function _get_user_quota_usage() {
     local username="$1"
     local lustre_path="$2"
 
-    lfs quota -q -u "$username" "$lustre_path" \
-        |tr -d '\n*' \
-        |awk '{print $2}'
+    if [[ -z "$global_calc_usage" ]]; then
+        lfs quota -q -u "$username" "$lustre_path" \
+            | tr -d '\n*' \
+            | awk '{print $2}'
+    else
+        lfs quota -v -q -u "$username" "$lustre_path" 2>/dev/null \
+            | grep '^\s\+[0-9]\+\*\?\s' \
+            | awk '{ sum+=$1 } END { print sum }'
```

Comment:
As mentioned in the PR comments: you can combine the `grep` and the `awk` here like this:
```
awk '/^\s+[0-9]+*?\s/ { sum +=$1 } END { print sum }'
```